src/main/java/core/basesyntax/service/impl/ParseDataImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParseData;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParseDataImpl implements ParseData {
+    private static final String SEPARATOR = ",";
+    private static final int INDEX_ZERO = 0;
+    private static final int INDEX_ONE = 1;
+    private static final int INDEX_TWO = 2;

```

Comment:
it's not an informative naming, name after roles of constants, not their values, which can change. 
Let's name them OPERATION_INDEX, FRUIT_INDEX, AMOUNT_INDEX or smth along those lines